docs/docs.adoc

```diff
@@ -0,0 +1,274 @@
+= Trostani : User Documentation
+
+:toc:
+
+== Overview
+
+Trostani is a __Discord__ bot with some **Magic the Gathering** features.
+
+== Commands
+
+Every commands starts with a prefix, by default this prefix is set to `$`
+
+=== Tnmt
+
+`tnmt` command is used to interact with Challonge and **organize tournaments**
+
+==== Subcommands
+
+In order to keep things clear, some commands are restricted to one channel only.
+
+The create subcommands can only be send in a __specified channel__ grouping all tournaments announcements.
+
+When a tournament is created, a __dedicated channel__ is created for **this tournament only**.
+
+===== Administrators
+
+**create** : available only in __specified channel__
+
+`create` subcommand is used to create a tournament on Challonge, this will trigger the __dedicated channel__ creation.
+
+```text
+$tnmt create <name> // <description> // <type> (SW, DE, SE or RR) // <format> // <date> (optional, format: YYYY-MM-DD at HH:MM)
+```
+
+Example :
+
+```text
+$tnmt create Tournament of The Year // A tournament organized using Trostani // SW // Standard // 2020-11-07 at 17:00
+```
+
+**start** : available in __dedicated channel__
+
+`start` subcommand is used to start tournament **identified by channel name**
+
+```text
+$tnmt start
+```
+
+**finalize** : available in __dedicated channel__
+
+`finalize` subcommand is used to finish and close tournament **identified by channel name**
+
+```text
+$tnmt finalize
+```
+
+===== Players
+
+**list** : no channel restrictions
+
+`list` subcommand is used to list tournaments from Challonge
+
+```text
+$tnmt list <status> (optional)
+```
+
+Possible values for status are :
+
+- pending
+- underway
+- complete
+
+Example :
+
+```text
+$tnmt list pending
+```
+
+**join** : available in __dedicated channel__
+
+`join` subcommand is used to join a tournament, you can add an optional info string like archetype of your deck
+
+```text
+$tnmt join <archetype> (optional)
+```
+
+Example :
+
+```text
+$tnmt join Izzet Drake
+Deck
+4 Falaises des eaux vives (M20) 252
```

Comment:
This should use english card name

Reply:
Fixed by https://github.com/papey/trostani/commit/d6cfe4c7b51d10229ca62a75ef564b4b2aac264c